elab_bridge/tests/test_server_interface.py

```diff
@@ -43,6 +43,9 @@ def test_download_experiment(initialize_test_dir):
                                      experiment_id=experiment_id,
                                      format='json')
 
+    if len(upload) != len(experiment):
+        assert False
```

Comment:
This is not a good way of stating it. An assert statement is already checking for a condition and raising an error if the condition is not met. Here you are double conditioning... This would be a better way to write it 

```suggestion
    assert len(upload) == len(experiment)
```